### PR TITLE
publish v1.0.0-dev.9

### PR DESCRIPTION
**Pull request by dshukertjr**
fixes https://github.com/supabase-community/supabase-flutter/issues/221
fixes https://github.com/supabase-community/supabase-flutter/issues/215
fixes https://github.com/supabase-community/supabase-flutter/issues/225

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## [1.0.0-dev.9]
+
+- fix: update supabase to [v1.0.0-dev.8](https://github.com/supabase-community/supabase-dart/pull/146)
```
Comment (Vinzent03) [inline CHANGELOG.md:3]: Is there a reason of linking to the pr here, but linking to the changelog at the realtime update in line 7?

Comment (dshukertjr) [reply CHANGELOG.md:3]: I haven't been consistent with this, but I am trying to write the changelog in the following format:

- supabase_flutter update
- update supabase version
  - contents of the supabase version update
  - update sub-library version 
    - contents of the sub-library version update

Comment (dshukertjr) [reply CHANGELOG.md]: What do you think about this format? Too complicated?

Comment (dshukertjr) [reply CHANGELOG.md]: What do you think about this format? Too complicated?

Comment (Vinzent03) [reply CHANGELOG.md]: I like that you don't have to read every's package changelog individually, but you link to the pr on supabase update, but to the changelog on realtimee update.

Comment (dshukertjr) [reply CHANGELOG.md]: Ah, I see. Nice catch!

**Review by Vinzent03 — APPROVED** [review]

**Pull request merged**